### ISSUE TITLE
🐛 Bug Report: GIF files retrieved from buckets are too small

### ISSUE THREAD

**Issue by MooseSaeed** [appwrite/appwrite#3183]
### 👟 Reproduction steps

When I try to retrieve a specific gif file from a bucket it returns too small to be seen.

```
appwrite.storage.getFilePreview(
            bucketID,
            file.$id
          );
```

Any other file such as jpg and png works just fine.

Just try out to save a gif image file in a bucket and use appwrite's get file preview with web SDK and the retrieved gif file will be too small and very low resolution.

### 👍 Expected behavior

It should return with the defaults mentioned in docs which is full size.

I have checked the console and the `href` there for the gif works in browser perfectly fine while the `href` retrieved from get file preview is misbehaving. the only difference between the two `hrefs` is that in the console it adds `&mode=admin` after the `href` and replaces `preview?` with `view` 

This is the `href` that works fine in appwrite's console:

`http://localhost/v1/storage/buckets/testing-out-a-component/files/testing-out-a-component1/view?project=tailwinder&mode=admin`

This is the href that is retrieved from get file preview using appwrite web SDK which is too small gifs (it works fine on jpg and png):

`http://localhost/v1/storage/buckets/testing-out-a-component/files/testing-out-a-component2/preview?project=tailwinder`

### 👎 Actual Behavior

The GIF returned very slow and impossible to see. In order to see it I have to zoom like 500% in browser.

### 🎲 Appwrite version

Version 0.13.x

### 💻 Operating system

Windows

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by MooseSaeed** [comment]
That's weird! when I use get file preview on jpg or png works just fine, but when I use it with any of my gifs it returns too small to be seen.

This is an example gif:
![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4b4f4f4a6c6270434c796d7044454569766f2f67697068792e676966](https://user-images.githubusercontent.com/94674993/167175126-736eeb9c-edc4-4429-9dc1-0026eace84d9.gif)

**Comment by Meldiron** [comment]
Thanks for the GIF you provided. I can see that preview of this fike takes A LOT of time, that's probably why there is a problem regarding rendering.

Ill keep investigating. In the meantime, you can use `view` instead of `preview`endpoint.

```
https://matej.appwrite.org/v1/storage/buckets/s1/files/6278ac328986f80916f1/view?project=BigInt
```

Regarding SDK, that should be `getFileView` or `getFileDownload` depending on what type of data you expect (url/stream)

**Comment by Meldiron** [comment]
Addressed in a PR: https://github.com/utopia-php/image/pull/20

**Comment by Meldiron** [comment]
PR merged. The issue should be addressed in 0.14. Feel free to reopen if it does not work for you.
